rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -301,7 +354,7 @@ class Player : public rclcpp::Node
   ROSBAG2_TRANSPORT_PUBLIC
   /// \brief Getter for the currently stored storage options
   /// \return Copy of the currently stored storage options
-  const rosbag2_storage::StorageOptions & get_storage_options();
+  std::vector<rosbag2_storage::StorageOptions> get_storage_options();
```

Comment:
@christophebedard If we add a new API instead of changing the existent `get_storage_options(),` this PR will likely become API/ABI compatible, and we will be able to backport it to Jazzy.

The idea for the name: `get_all_storage_options()`

Reply:
I've changed it back to `const rosbag2_storage::StorageOptions & get_storage_options()` and added `std::vector<rosbag2_storage::StorageOptions> get_all_storage_options()`. I marked `get_storage_options()` as deprecated, but we can leave that out when backporting to Jazzy.

38403090993b817b3cdc8d91abcb9f8e94d3e166

Reply:
Thanks!

Reply:
I removed the deprecation notice in b22e27b73009b24cd3ae97064b203c838922e698. This way we can just backport this PR to Jazzy easily. I'll do the deprecation in another PR for Rolling, since we might want to deprecate other things too.